scripts/intertidal_abund_me.py

```diff
@@ -14,8 +15,12 @@ def __init__(self):
                         retriever_minimum_version='2.0.dev',
                         version='1.4.0',
                         urls={"main": "https://ndownloader.figshare.com/files/5600831"},
-                        tables={"main": Table("main", cleanup=Cleanup(correct_invalid_value, nulls=[-999.9]))}
+                        tables={"main": Table("main", cleanup=Cleanup(correct_invalid_value, missingValues=[-999.9]))}
```

Comment:
we can clean up this file to a structure like 
```
class main(Script):
    def __init__(self, **kwargs):
        Script.__init__(self, **kwargs)
        self.name =
```
so that we take care of this clean up function like we have done in other files
`cleanup=Cleanup(correct_invalid_value, missingValues=[-999.9]))}`